lib/tether/tether_linux.go

```diff
@@ -75,18 +76,17 @@ func (t *tether) childReaper() error {
 				// general resiliency
 				defer func() {
 					if r := recover(); r != nil {
-						fmt.Printf("Recovered in childReaper %v", r)
+						fmt.Printf("Recovered in childReaper %s", debug.Stack())
```

Comment:
Write this to os.Stderr - that's redirected to the debug log, but stdout is not.